third_party/terraform/resources/resource_dataflow_flex_template_job.go.erb

```diff
@@ -115,8 +123,11 @@ func resourceDataflowFlexTemplateJobCreate(d *schema.ResourceData, meta interfac
 		return err
 	}
 
+	zone, _ := getZone(d, config)
```

Comment:
This field is optional in the API, and that's why we can ignore the error right? Mind commenting as much?

Reply:
Done.